apps/frontend/core-dapp/src/features/ppo/stake/StakeStore.ts

```diff
@@ -25,15 +25,15 @@ export class StakeStore {
     this.showDelegate = show
   }
 
-  stake(): {
+  stake(): Promise<{
     success: boolean
     error?: string | undefined
-  } {
+  }> {
     const balance = this.getBalance()
     if (this.currentStakingValue > balance) {
```

Comment:
Maybe we could update this code to just do this below, and delete the `getBalance` method, this looks really redundant, it's reassigning the same thing to different variables twice while there's essentially no complexity at all, (token is "assigned" to getBalance, then getBalance is called to assign to `balance`, when we could've just used `tokenBalance` directly) understand that it's not changes from this PR, maybe good to have it fixed since it's relevant

Also the existing code is somewhat wrong, ppoTokenStore cannot be undefined, and `tokenBalance` is deprecated

```suggestion
    const { tokenBalanceFormat } = this.root.ppoTokenStore
    if (this.currentStakingValue > +(tokenBalanceFormat ?? 0)) {
```